### ISSUE TITLE
Module does not check for different formatting than default one

### ISSUE THREAD

**Issue by JeanneD4RK** [juliangruber/parse-apache-directory-index#15]
Hi

Nice module, however it does not check for the config of the apache directory, which greatly changes the page structure depending on the options. Eg : options C=N;O=A removes <table> and <tr> tags.

https://httpd.apache.org/docs/2.4/en/mod/mod_autoindex.html

**Comment by juliangruber** [comment]
Hey, since we can't read the apache config, we'd need to change the DOM parsing to be more flexible. PRs welcome!

**Comment by JeanneD4RK** [comment]
Your lib works with Format type #2.
Another solution would be to check for F= (format argument) in the URL, replace it's value with F=2, or append it if it's not present, to make sure the directory is listed as expected. 

Eg : https://myserver.lan/dir -> https://myserver.lan/dir?F=2 or https://myserver.lan/dir?F=0 -> https://myserver.lan/dir?F=2

**Comment by juliangruber** [comment]
ah! If that is part of the URL then yeah let's just rewrite it

**Comment by JeanneD4RK** [comment]
Yes, this can be set by a GET param, this also overwrites the server side config.
